.github/workflows/continuous_integration.yml

```diff
@@ -0,0 +1,42 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  build:
+    name: build
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        django_version: ["2.2"]
+        python_version: ["3.6", "3.7", "3.8"]
+        variant: [alpine, slim]
+        include:
+          - variant: alpine
+            pg_major: 12
+            pg_version: 12.2-r0
+          - variant: slim
+            pg_major: 10
+            pg_version: 10.13-1.pgdg100+1
+    env:
+      CI: 1
```

Comment:
`CI` is always set to `true`.

See: https://help.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables#default-environment-variables

Reply:
Oh fabulous, thank you! I'll submit a small PR to update this in [docker-tippecanoe](https://github.com/azavea/docker-tippecanoe/blob/master/.github/workflows/continuous-integration.yml#L19) as well.